### PR TITLE
Update LocatorStorageService.cs

### PR DESCRIPTION
**Pull request by Oleg-Mozhey**

### CODE REVIEW

**Review by andrey-brovko — APPROVED** [review]

File: Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs [hunk 1]
```diff
@@ -12,12 +13,24 @@ public class LocatorStorageService
...
+            .Where(p => type.IsAssignableFrom(p) && p.IsClass);
 
-        var element = (T)mainPage.GetField(elementName.ToCamelCase())?.GetValue(page)!;
-        if (element is null) throw new Exception($"'{elementName}' selectors not found.");
-        return element;
+        IWebElement element = null;
+        foreach (var pageType in types)
+        {
+            var pageTemp = Activator.CreateInstance(pageType);
+            var temp = (IWebElement) pageType.GetField(elementName.ToCamelCase())?.GetValue(pageTemp)!;
```
Comment (vladimir-s-quantori) [inline Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs:25]: `pageTemp` and `temp` variable names are not very clear, is there any other names you can come up with?

Comment (Oleg-Mozhey) [reply Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs]: Changed pageTemp to instanceOfClassWithElements
Changed temp to classField

Added checking for empty element name:
        if (elementName.Equals("")) throw new Exception("element name can not be empty. Please correct test step");

File: Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs [hunk 2]
```diff
@@ -13,24 +13,23 @@ public class LocatorStorageService
     //TODO: Impl factory
     public T Get<T>(string elementName)
     {
+        if (elementName.Equals("")) throw new Exception("element name can not be empty. Please correct test step");
```
Comment (vladimir-s-quantori) [inline Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs:16]: It's not the best way to check string emptiness like this.
1. Better use `string.IsNullOrEmpty(elementName)`
2. There is no need to create new string with empty value `""`, you can use `string.Empty` for this case

Comment (Oleg-Mozhey) [reply Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs]: Changed to string.IsNullOrEmpty(elementName)

Thanks!

**Review by vladimir-s-quantori — APPROVED** [review]

**Pull request merged**